projects/plugins/jetpack/modules/notes.php

```diff
@@ -173,7 +173,7 @@ public function admin_bar_menu() {
 			return;
 		}
 
-		$wpcom_locale = get_locale();
+		$user_locale = get_user_locale();
```

Comment:
On a Simple Classic site (and later also Simple Default sites via the Site-Level User Profile project), we can set the language in Users -> Profile. That means `get_user_local()` could be different from `/me/account` language.

I think this is OK because the rest of the admin bar menus will use that language anyway, but wanted to note it here. What do you think / could you double check?

Reply:
I created this PR for Simple sites and I believe this will catch the Site-Level User profile project. WDYT?

Reply:
> On a Simple Classic site (and later also Simple Default sites via the Site-Level User Profile project), we can set the language in Users -> Profile. That means get_user_locale() could be different from /me/account language.

That's the main idea of the PR, show the notification in the profile selected locale, as admin menu does.

Reply:
That makes sense! Haven't actually tested but the code (both this and wpcom side) look good!

Reply:
I just updated converting the locale to ISO 639-1 to make sure no language files are missed.